southpark-incremental-indexing/tests/distributed/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   flow:
     image: south
     build:
-      context: .
+      context: ../../../southpark-incremental-indexing/tests/distributed
```

Comment:
is it an intention to adapt the old test with the `incremental indexing` one? I would keep it as it was? I'd try to avoid having this dependency

Reply:
Yeah, that's what I thought initially. But if we don't want that, I can remove it (or maybe we could add it later as a separate PR to keep this one to the point)

Reply:
let's remove it for now and maybe add in a separate PR if needed